src/patternfly/layouts/Bullseye/styles.scss

```diff
@@ -1,8 +1,18 @@
+@import "../../patternfly-utilities";
+
 .pf-l-bullseye {
+  $pf-l-gallery--Padding: 0 !default;
```

Comment:
I am not following:
1- why are gallery variables in the bullseye layout?
1- Do we need a padding here, what's it for?
1- why are these variables within the {}?

Reply:
@andresgalante good questions.

1) Good catch. I need to rename that.

2) We may or may not need the padding. I can think of scenarios where it would be necessary and some where it wouldn't. I would say that is ultimately going to be a design question. Right now they are just placeholders, with an added bonus of providing some consistency of code across the layouts. As a general rule I like setting the padding and margin explicitly because we don't always know what elements the class will be applied to. I can see scenarios (especially with layouts) where the elements will be using a `<ul>` or something similar, so in the spirit of modularity I set padding and margin.

3) Putting them inside of the selector helps keep them scoped to that selector. If I place them outside of the selector, it will put them in the global scope, which can have unpredictable results down the road and ultimately can lead to maintainability issues. With all that being said, do you think there Is benefit to placing them outside of the selector?